docs/conf.py

```diff
@@ -84,6 +85,10 @@
     ),
 }
 
+# Setting pandas option for the examples
+pd.set_option("mode.copy_on_write", True)
+
+
```

Comment:
I think we can remove this.

Reply:
removed in 6160ac9079c0a8818c656dc241df90dc4a25a9f9